crates/kornia-imgproc/src/features.rs

```diff
@@ -108,6 +110,9 @@ impl HarrisResponse {
             k: 0.04,
             grads_mode: GradsMode::default(),
             sigmas: 0.0,
+            dx2_data: Vec::new(),
```

Comment:
probably the right thing is to pass ImageSize in the constructor to avoid a resize